### ISSUE TITLE
Use secure mode on UI not work

### ISSUE THREAD

**Issue by mrpalide** [skycoin/skywire#979]
**Describe the bug**
*Use secure mode* checkbox on vpn-server setting on UI not work.

**Environment information:**

- OS: Linux
- Platform: Kubuntu 20.04

**Steps to Reproduce**
On UI, change *Use secure mode* checkbox on vpn-server setting. Refresh page or check config file, that value of that not changed.

**Actual behavior**
Nothing.

**Expected behavior**
We need a toggle mode to change it from true to false, and vice versa.

**Comment by Senyoret1** [comment]
I tested and I think the first time I tried the error was present, but in all the tries I made after that it worked well. Not sure if it failed the first time I tried because I checked the value before the data was refreshed in the UI or if there really was a bug. In any case, I was unable to reproduce it again and, if there is a bug, it appear that it is not that easy to debug. Maybe someone more familiar with the back-end than me should take a look
